estela-api/api/views/cronjob.py

```diff
@@ -115,6 +116,20 @@ def update(self, request, *args, **kwargs):
 
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+    @swagger_auto_schema(
+        responses={status.HTTP_204_NO_CONTENT: "Cronjob deleted"},
+    )
+    def destroy(self, request, *args, **kwargs):
+        instance = self.get_object()
+        self.perform_destroy(instance)
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+    def perform_destroy(self, instance):
+        instance.status = SpiderCronJob.DISABLED_STATUS
+        delete_cronjob(instance.name)
```

Comment:
I think is better disable instead of delete.

Reply:
Done